packages/story-editor/src/app/highlights/quickActions/useQuickActions.js

```diff
@@ -474,7 +474,12 @@ const useQuickActions = () => {
 
   const isVideoTrimEnabled = useFeature('enableVideoTrim');
   const videoCommonActions = useMemo(() => {
-    return isVideoTrimEnabled
+    if (!selectedElement?.resource) {
+      return [];
+    }
+    const { resource } = selectedElement;
+    const { isMuting, isTranscoding, isTrimming } = resource;
+    return isVideoTrimEnabled && !isMuting && !isTranscoding && !isTrimming
```

Comment:
This conditional logic stops a user from transcoding, muting and trimming all at the same time.

Reply:
This is fixed in #8944 - `isVideoTrimEnabled` will be false if the video is already transcoding.